asu/openapi.yml

```diff
@@ -265,6 +265,13 @@ components:
             Ability to specify filesystem running on device. Attaching this
             optional parameter will limit the ImageBuilder to only build
             firmware with that filesystem.
+        rootfssize:
```

Comment:
Please specify a maximum, see the specific parameter names [here](https://swagger.io/docs/specification/data-models/data-types/#numbers)

Reply:
Shouldn't the maximum be configurable by the server operator? I mainly thought to add it to prevent an asu server from building an image that would take up too many resources

Reply:
@aparcar I've done so in the latest commit however I'm still not sure how to have it configurable via openapi.yml